### PR TITLE
Add `validation-dir` parameter to lint

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Add `validation-dir` parameter to lint  
  [dnkoutso](https://github.com/dnkoutso)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: lib/cocoapods/validator.rb [hunk 1]
```diff
@@ -58,7 +58,6 @@ def initialize(spec_or_path, source_urls, platforms = [])
         end
         result
       end
-      @use_frameworks = true
```
Comment (dnkoutso) [inline lib/cocoapods/validator.rb:61]: this is also setup above, expand a few lines and you will see it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Add ability to specify the `validation-dir` during `lint`.
```
Comment (amorde) [inline CHANGELOG.md:11]: super nit: this has 4 spaces at the end instead of 2

Comment (dnkoutso) [reply CHANGELOG.md:11]: fixed it in a post commit.

**Pull request merged**